client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -110,7 +110,7 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
             'amendment_paragraphs',
             'modified_final_version'
         ]);
-        const amendmentFields: (keyof Motion)[] = ['amendment_paragraphs'];
+        const amendmentFields: (keyof Motion)[] = listFields.concat(['amendment_paragraphs', "lead_motion_id", "amendment_ids"]);
```

Comment:
You need both id fields, so the client knows, which are amendments ;) This was the "biggest" change.